src/stories/30_Monetization/Publisher_Served.html

```diff
@@ -15,10 +15,10 @@
 <html ⚡ lang="en">
 <head>
   <meta charset="utf-8">
-  <link rel="canonical" href="https://ampbyexample.com/stories/monetization/amp-story-auto-ads/">
+  <link rel="canonical" href="https://ampbyexample.com/stories/monetization/Publisher_Served/">
   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
   <script async src="https://cdn.ampproject.org/v0.js"></script>
-  <title>amp-story-auto-ads</title>
+  <title>Publisher Served</title>
```

Comment:
Same as above,  "Publisher served Ads for AMP Stories"

Reply:
done.